pkg/attestation/azure/azure.go

```diff
@@ -0,0 +1,33 @@
+// Copyright (c) Ultraviolet
+// SPDX-License-Identifier: Apache-2.0
+
+package azure
+
+import (
+	"github.com/edgelesssys/go-azguestattestation/maa"
+)
+
+var (
+	OSBuild  = "UVC"
+	OSType   = "Linux"
+	OSDistro = "UVC"
+	MaaURL   = "https://sharedeus.eus.attest.azure.net"
+)
```

Comment:
use a struct

Reply:
and a struct receiver in the function they are required, this is the idiomatic way for go